<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OpenCarOrderCardViewModel</class>
 <widget class="QWidget" name="OpenCarOrderCardViewModel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>740</width>
    <height>462</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="3" column="1">
      <widget class="QLabel" name="price">
       <property name="text">
        <string>90000 рублей</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="deadlineDate">
       <property name="text">
        <string>18/10/2024, до 12 часов</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Оставшееся время аренды</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Отвезти в офис к</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="carOrderId">
       <property name="text">
        <string>33efb94c-1a27-45c6-9e53-7c31677ea35e</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Стоимость аренды</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="remainLeaseDateTime">
       <property name="text">
        <string>2 Дня, 12 часов, 35 минут</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout" stretch="7,2,2">
     <item>
      <widget class="QLabel" name="carImage">
       <property name="text">
        <string>CarImage</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>UID-автомобиля</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="carId">
         <property name="text">
          <string>33efb94c-1a27-45c6-9e53-7c31677ea35e</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="carName">
       <property name="text">
        <string>CarName</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
